bin/installHugo.sh

```diff
@@ -0,0 +1,6 @@
+#/bin/bash
+set -euo pipefail
```

Comment:
I have an error with this option: 
```
➜  ps-docs git:(master) ✗ ./installHugo.sh
./installHugo.sh: 2: set: Illegal option -o pipefail
```

Reply:
Are you sure you're running with bash, and not sh?
Because I can only reproduce under sh context, not even on Bash, Zsh, or anything else :/

Reply:
But you locate something false, the shebang isn't good:
`#/bin/bash` must be `#!/bin/bash` That's maybe why you have some problem. (Your configuration maybe use sh by default)

Reply:
So the shebang needs to be updated, am I right?

Reply:
Yep!